ocaml/idl/datamodel_pool.ml

```diff
@@ -627,7 +627,14 @@ let install_ca_certificate =
 let certificate_uninstall =
   call ~name:"certificate_uninstall"
     ~doc:"Remove a pool-wide TLS CA certificate."
-    ~params:[(String, "name", "The certificate name")]
+    ~params:
+      [
+        (String, "name", "The certificate name")
+      ; ( Bool
+        , "force"
+        , "If true, remove the DB entry even if the file is non-existent"
```

Comment:
This call is deprecated, please don't modify it. (we have to change the lifecycle, but the last entry is the deprecation!)